scss/components/popover.scss

```diff
@@ -170,6 +170,10 @@ $block: #{$fd-namespace}-popover;
         box-shadow: 0 5px 20px 0 var(--fd-color-neutral-3), 0 2px 8px 0 var(--fd-color-neutral-2);
         opacity: 1;
 
+        &--full-width {
+          width: 100%;
+      }
+
```

Comment:
I don't think there is a need for this here.  Because popper.js-based popovers are placed in a different place in the DOM (child of body), setting the width to 100% will just span the entire body rather than matching its size to its reference element.  Sizing of the popover in relation to the reference element is going to be the responsibility of the implementation library.

Reply:
Ooof I thought about this wrong! Good call